lib/p2p/packets/utils.ts

```diff
@@ -18,10 +19,31 @@ export const convertNodeState = (nodeState: pb.NodeState.AsObject) => {
     addresses: nodeState.addressesList,
     raidenAddress: nodeState.raidenAddress,
     lndPubKeys: convertKvpArrayToKvps(nodeState.lndPubKeysMap),
+    lndUris: convertLndUris(nodeState.lndUrisMap),
     tokenIdentifiers: convertKvpArrayToKvps(nodeState.tokenIdentifiersMap),
   });
 };
 
+const convertLndUris = <T>(kvpArray: [string, T][]): { [key: string]: T } => {
```

Comment:
Why the generic `<T>` here? Is this in case the type of `LndUris` changes in the future from a repeated string? I figure we can just replace `T` with `string[]` right? Would that also eliminate the need for `@ts-ignore` below?

Reply:
Thanks for noticing. I meant to do that, but somehow missed this. I've removed the `@ts-ignore` and replaced `T` with `string[]`.